sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -828,6 +828,8 @@ object GpuOverrides extends Logging {
       sparkSig = (TypeSig.cpuAtomics + TypeSig.STRUCT + TypeSig.ARRAY + TypeSig.MAP +
         TypeSig.UDT).nested())))
 
+  val fileFormats = basicFormats ++ ShimLoader.getSparkShims.getFileFormats
```

Comment:
nit: The original `fileFormats` was lazy before. Not 100% sure why we did it that way, but it would be nice to mark both of them as lazy again for consistency.

Reply:
Updated